### ISSUE TITLE
[MSFT-113] Member registration page: Errors and alerts are not being announced for screen reader users.

### ISSUE THREAD

**Issue by veshu** [importtest/importtest#116]
**Steps to Reproduce**
1. Go to the Members registration page:https://accounts.topcoder.com/member/registration. 2. Ensure that the screen reader is switched on. 3. Press tab key till your arrive at the user name field' 4. Type a single character in the field. 6. Since one of the criteria for user name is to have a user name between 2-15 characters, an error is displayed. 7. This error is not announced by the screen readers.

**Expected Result**
Screen reader should immediately announce the error message so that the user corrects the field before proceeding.

**Actual Result**
Errors and alerts are not being announced by the screen readers. Screen readers are silent hence users cannot understand the activity on the screen.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.3.1 Error Identification
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2018
